pages/linux/iostat.md

```diff
@@ -1,19 +1,23 @@
 # iostat
 
-> Report stats for devices and partitions.
+> Report statistics for devices and partitions.
 
-- Display disk statistics with disk IDs in human readable format:
+- Display a report of CPU and disk statistics since system startup:
 
-`iostat -h`
-
-- Display disk statistics with disk names (including LVM) in human readable format:
-
-`iostat -Nh`
+`iostat`
 
 - Display CPU statistics:
 
 `iostat -c`
 
-- Display extended disk statistics with disk names:
+- Display disk statistics with disk names (including LVM):
+
+`iostat -N`
+
+- Display extended disk statistics with disk names for device "sda":
+
+`iostat -xN {{sda}}`
+
+- Display incremental reports of CPU and disk statistics every "interval" second(s) with units converted to megabytes:
 
-`iostat -xN`
+`iostat -m {{interval}}`
```

Comment:
Perhaps a specific example (like "sda" above) would make the description easier to read -- e.g., "every 5 seconds". 

Also, does the -m take the parameter or is it independent (i.e., for the megabytes thing)? If it's the latter, having these together may give out the wrong idea.